jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -380,6 +379,51 @@ with what features/services are supported.
             <reference id="serverless.application.deploy"/>
             <separator/>
         </group>
+
+        <group id="aws.toolkit.s3viewer.transferActions">
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.DownloadObjectAction"/>
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.UploadObjectAction"/>
+        </group>
+
+        <group id="aws.toolkit.s3viewer.versionActions">
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.ViewObjectVersionAction"/>
+        </group>
+
+        <group id="aws.toolkit.s3viewer.mutateActions">
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.NewFolderAction"/>
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.RenameObjectAction" use-shortcut-of="RenameElement"/>
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.DeleteObjectAction" use-shortcut-of="$Delete"/>
+        </group>
+
+        <group id="aws.toolkit.s3viewer.miscActions">
+            <action class="software.aws.toolkits.jetbrains.services.s3.objectActions.RefreshTreeAction"/>
```

Comment:
`use-shortcut-of="Refresh"`?

Reply:
Ah we didnt have that before so didn't think to add it, good idea